kmk/extensions/oled.py

```diff
@@ -0,0 +1,291 @@
+import busio
+import gc
+from supervisor import ticks_ms
+
+import adafruit_displayio_ssd1306
+import displayio
+import terminalio
+from adafruit_display_text import label
+
+from kmk.extensions import Extension
+from kmk.handlers.stock import passthrough as handler_passthrough
+from kmk.keys import make_key
+from kmk.kmktime import check_deadline
+
+
+class OledEntryType:
+    TXT = 0
+    IMG = 1
+
+
+class OledData:
+    def __init__(
+        self,
+        entries=None,
+    ):
+        if entries is not None:
+            self.data = entries
+
+    @staticmethod
+    def oled_text_entry(
+        text='',
+        x=0,
+        y=0,
+        x_anchor='left',
+        y_anchor='top',
+        direction='LTR',
+        line_spacing=0.75,
+        inverted=False,
+        layer=None,
+        side=None,
+    ):
+        _x = 0
+        _x_anchor = 0.0
+        _y_anchor = 0.0
+        if x_anchor == 'left' or x_anchor == 'L':
```

Comment:
I'd recommend to stick to just single letters, saves space and code, and matches the `direction` option from the adafruit lib.

Reply:
Something left from the earlier iteration that I thought won't hurt. Removing that leaving only single letters.